### PR TITLE
Update the web console to request k8s api on v1beta3

### PR DESCRIPTION
**Pull request by jwforres**

### CODE REVIEW

**Comment by jwforres** [comment]
@liggitt 

pending OK from @smarterclayton that v1beta3 is stabilized enough to deliver these changes

**Comment by jwforres** [comment]
also will need to make sure either https://github.com/openshift/origin/pull/954 is updated and merges after this, or that this PR also makes any changes related to pulling and reading from quotas and limitRanges

**Comment by jwforres** [comment]
@csrwng FYI since this could affect your create from template stuff

**Comment by jwforres** [comment]
@liggitt got the OK from clayton, review please and then i'll generate bindata

File: assets/app/scripts/services/labelFilter.js [hunk 1]
```diff
@@ -146,12 +146,7 @@ angular.module('openshiftConsole')
   };
 
   LabelFilter.prototype._extractLabelsFromItem = function(item, map) {
-    // TODO temporary until we handle everything with item.metadata.labels
-    var labels = item.labels;
-    if (item.metadata) {
-      labels = item.metadata.labels;
-    }
-
+    var labels = item.metadata.labels;
```
Comment (liggitt) [inline assets/app/scripts/services/labelFilter.js]: should we check if item.metadata is defined?

**Comment by liggitt** [comment]
LGTM otherwise

**Comment by jwforres** [comment]
[merge]

**Pull request merged**